safe_control_gym/hyperparameters/hpo_vizier.py

```diff
@@ -0,0 +1,429 @@
+""" The implementation of HPO class using Vizier
+
+Reference:
+    * https://oss-vizier.readthedocs.io/en/latest/
+    * https://arxiv.org/pdf/0912.3995
+
+"""
+
+from vizier import service
+from vizier.service import clients
+from vizier.service import pyvizier as vz
+from vizier.service import servers
+
+import os, time
+from copy import deepcopy
+from functools import partial
+import numpy as np
+import yaml, csv
+import matplotlib.pyplot as plt
+
+from safe_control_gym.hyperparameters.hpo_sampler import HYPERPARAMS_DICT
+from safe_control_gym.experiments.base_experiment import BaseExperiment
+from safe_control_gym.utils.logging import ExperimentLogger
+from safe_control_gym.utils.registration import make
+from safe_control_gym.utils.utils import mkdirs
+
+class HPO(object):
```

Comment:
This class is defined in both hpo_optuna and hpo_vizier, naming should be more precise instead of having the same class defined in two places.

Reply:
Done